test/parallel/test-cluster-inspector-debug-port.js

```diff
@@ -4,7 +4,7 @@ const common = require('../common');
 common.skipIfInspectorDisabled();
 const assert = require('assert');
 const cluster = require('cluster');
-const debuggerPort = common.PORT;
+const debuggerPort = 0;
```

Comment:
This is going to result in using ports `1`, `2` etc. It will never result in using port `0` I don't think. Does this actually pass if run without superuser privileges??!!  I suspect the purpose of this test is to test passing valid port numbers to various flags, so I think `common.PORT` is really the most correct thing to use here. This test should probably be moved (unmodified) to `test/sequential`.